src/pipeline_v3/tests/unit/test_index_manager.py

```
@@ -4,20 +4,16 @@
 import uuid
-from pathlib import Path
 from unittest.mock import Mock, patch
 
 import pytest
 
-sys.path.insert(0, str(Path(__file__).parent.parent.parent))
-
-from core.index_manager import IndexManager
-from core.registry import DocumentState, IndexType
+from src.pipeline_v3.core.index_manager import IndexManager
+from src.pipeline_v3.core.registry import DocumentState, IndexType
```

Ruff (0.11.9)

`src.pipeline_v3.core.registry.DocumentState` imported but unused

Remove unused import: `src.pipeline_v3.core.registry.DocumentState`

(F401)